backend/src/apiserver/main.go

```diff
@@ -104,20 +114,14 @@ func main() {
 
 	ctrllog.SetLogger(zap.New(zap.UseFlagOptions(&zap.Options{Level: zapLevel})))
 
-	clientManager, err := cm.NewClientManager()
+	clientManager, err := cm.NewClientManager(options)
 	if err != nil {
 		glog.Fatalf("Failed to initialize ClientManager: %v", err)
 	}
+	webhookOnlyMode := common.IsOnlyKubernetesWebhookMode()
```

Comment:
This should be after the `defer clientManager.Close()` to ensure if this function panics, the defer will be called.